templates/akeneo/files/.platform.app.yaml

```diff
@@ -0,0 +1,102 @@
+# This file describes an application.
+
+# The name of this app. Must be unique within a project.
+name: akeneo
```

Comment:
Template applications should always be named `app`.  (That will need to be updated in routes.yaml, too.)

Reply:
changed it to `app` :)